share/migrations/0039_update_trigger_migrations_20160906_1801.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
```

Comment:
You don't need to add a trigger update unless you change something that inherits from `ShareObject`